src/components/ArticleList.jsx

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+
+
+export default function ArticleList() {
+    const [articleList, setArticleList] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        axios
+            .get("https://nc-news-u90o.onrender.com/api/articles/") 
+            .then((response) => {
+                // console.log(response, "response");
```

Comment:
Make sure to remove console.logs and comments before sending for review

Reply:
Done